src/Interop/LlvmBindingsGenerator/Templates/T4/ExportsTemplate.cs

```diff
@@ -18,7 +18,7 @@ namespace LlvmBindingsGenerator.Templates
     /// Class to produce the template output
     /// </summary>
     
-    #line 1 "D:\GitHub\Ubiquity.NET\Ubiquity.NET.Llvm\src\Interop\LlvmBindingsGenerator\Templates\T4\ExportsTemplate.tt"
+    #line 1 "D:\github\Llvm.NET\src\Interop\LlvmBindingsGenerator\Templates\T4\ExportsTemplate.tt"
```

Comment:
Is this change to match the new build scripts?

Reply:
This file is generated by T4 (https://docs.microsoft.com/en-us/visualstudio/modeling/code-generation-and-t4-text-templates) and inherits line numbers from the system where they were generated. At some point it might be worth adding these files to the gitignore and then incorporating T4 generation into the build instead of relying on a developer opening the project in VS to regenerate them, but I didn’t find an easy way to do that, so deferred to the existing process.